### PR TITLE
Tweak the Pillar of the Paruns format

### PR DESCRIPTION
**Pull request by Susucre**
Tweaking the custom Pillar of the Paruns format.

After testing a few times, the following was modified:
-> I wanted to add a couple ability to Pillar of the Paruns. This was done here adding a custom version of the card, in an "Custom Alchemy" set. It is more straightforward than adding continuous effect to games in the custom game mode.
-> Starting life set to 25. Ideally this would be a custom option instead in the future, but I delayed working on that due to waiting a PR to be merged first.

image

### CODE REVIEW

File: Mage.Client/src/main/java/org/mage/plugins/card/dl/sources/ScryfallImageSupportCards.java [hunk 1]
```diff
@@ -533,6 +533,7 @@ public class ScryfallImageSupportCards {
             add("WOE"); // Wilds of Eldraine
             add("WOT"); // Wilds of Eldraine: Enchanting Tales
             add("WOC"); // Wilds of Eldraine Commander
+            add("CALC"); // Custom Alchemized versions of existing cards. Will download the original in directDownloadLinks
```
Comment (xenohedron) [inline Mage.Client/src/main/java/org/mage/plugins/card/dl/sources/ScryfallImageSupportCards.java:536]: It's slightly confusing to add a custom set in this list. But if that's the only way for it to work, it's okay.

File: Mage.Sets/src/mage/sets/CustomAlchemy.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import mage.cards.ExpansionSet;
+import mage.constants.Rarity;
+import mage.constants.SetType;
+
+/**
+ * Custom version of the official cards.
+ * Similar to Alchemy tweaks of cards.
+ * @author Susucr
+ */
+public final class CustomAlchemy extends ExpansionSet {
```
Comment (xenohedron) [inline Mage.Sets/src/mage/sets/CustomAlchemy.java:13]: Make sure this doesn't have boosters

File: Mage.Client/src/main/java/org/mage/plugins/card/dl/sources/ScryfallImageSupportCards.java (hunk 1 above)
Comment (Susucre) [reply Mage.Client/src/main/java/org/mage/plugins/card/dl/sources/ScryfallImageSupportCards.java:536]: Yeah that first add is for "attempt to download through scryfall", then the directDownloadLinks will direct url download the pillar image from dissension to the correct folder for CALC/1

I did not want to do an heavier rework of the logic, but still have some image on the custom card.

**Review by xenohedron — APPROVED** [review]
I'm okay with these workarounds- will give it a few days before merge in case anyone else has ideas for how to structure this more cleanly.

(I do eventually want to move custom cards into a different part of the file structure but since I don't have a proposal ready, not going to gate this, it'll be simple enough to move in the future.)

**Comment by Susucre** [comment]
And the main reason for the custom card and not adding more to the custom mode:
In long term, I'd like to phase out the custom mode entirely.
-> 6 starting hand and 25 life would be custom options available to any match/tournament.
-> The "skip first land drop" and "conjure a C-Pillar of the Paruns" would be on a custom Emblem each player would start with. But this needs for #10498 to be there first.

**Pull request merged**